app/src/main/res/values/strings.xml

```diff
@@ -49,19 +52,18 @@
     <string name="auto_updates_dialog_note">These settings can be changed later.</string>
 
     <string name="general">General</string>
-    <string name="general_description">General settings</string>
-    <string name="advanced">Advanced</string>
-    <string name="advanced_description">Advanced settings</string>
+    <string name="general_description">Theme, dynamic color</string>
     <string name="updates">Updates</string>
-    <string name="updates_description">Updates for ReVanced Manager</string>
-    <string name="opensource_licenses">Open source licenses</string>
-    <string name="opensource_licenses_description">View all the libraries used to make this application</string>
+    <string name="updates_description">Check for Update, changelogs</string>
     <string name="downloads">Downloads</string>
-    <string name="downloads_description">Manage downloaded content</string>
+    <string name="downloads_description">Split APK, downloaded application</string>
```

Comment:
This text is more accurate with the downloader plugin system branch.
```suggestion
    <string name="downloads_description">Downloader plugins and downloaded apps</string>
```

Reply:
How about "Downloaders and apps"?

Reply:
How about:

```suggestion
    <string name="downloads_description">Downloaders and apps</string>
```